VectorThrust.cs

```diff
@@ -223,25 +225,33 @@ public struct BA {
 
 	// Echo("Starting Main");
 	argument = argument.ToLower();
-	bool togglePower = argument.Contains(standbyArg.ToLower());
+	bool togglePower = argument.Contains(standbytogArg.ToLower());
 
 	bool anyArg =
 	argument.Contains(dampenersArg.ToLower()) ||
 	argument.Contains(cruiseArg.ToLower()) ||
 	argument.Contains(jetpackArg.ToLower()) ||
-	argument.Contains(standbyArg.ToLower()) ||
+	argument.Contains(standbytogArg.ToLower()) ||
 	argument.Contains(raiseAccelArg.ToLower()) ||
 	argument.Contains(lowerAccelArg.ToLower()) ||
 	argument.Contains(resetAccelArg.ToLower()) ||
 	argument.Contains(resetArg.ToLower()) ||
 	argument.Contains(applyTagsArg.ToLower()) ||
 	argument.Contains(removeTagsArg.ToLower());
 
-	// going into standby mode
-	if((togglePower && !standby) || goToStandby) {
+	// set standby mode on
+	if(argument.Contains(standbyonArg.ToLower()) || goToStandby) {
+		enterStandby();
+	        return;
+	// set standby mode off
+	} else if(argument.Contains(standbyoffArg.ToLlwer()) || comeFromStandby) {

```

Comment:
typo at ToLlwer()
this doesnt compile. you should try the code before submitting changes
```suggestion
	} else if(argument.Contains(standbyoffArg.ToLower()) || comeFromStandby) {
```

Reply:
no need to fix these, im modifying it slightly anyway. just thought i should let you know